Experimental/DistributedAuthoritySample/Assets/InputSystem/AvatarActions.inputactions.meta

```diff
@@ -8,7 +8,7 @@ ScriptedImporter:
   assetBundleName: 
   assetBundleVariant: 
   script: {fileID: 11500000, guid: 8404be70184654265930450def6a9037, type: 3}
-  generateWrapperCode: 0
+  generateWrapperCode: 1
```

Comment:
Actually, this can be reverted too. I'm seeing that the script file is being generated.